Libs/vtkSegmentationCore/vtkSegmentationModifier.cxx

```diff
@@ -198,6 +198,25 @@ void vtkSegmentationModifier::SeparateModifiedSegmentFromSharedLabelmap(vtkOrien
     }
 }
 
+inline int* GetMaximumExtent(const int *extent, vtkOrientedImageData* labelmap)
```

Comment:
```suggestion
//-----------------------------------------------------------------------------
inline int* GetMaximumExtent(const int *extent, vtkOrientedImageData* labelmap)
```

Reply:
I changed the function to a static protected method to be coherent with the call context.